tests/test_datacite_user_schema.py

```diff
@@ -0,0 +1,177 @@
+import pytest
+from datacite import schema40
+import jsonschema
+from osf.utils.metadata.schemas.utils import from_json
+
+@pytest.fixture()
+def datacite_user_schema():
+    return from_json('datacite-v4.0-osf-user-schema.json')
+
+# Valid Metadata
+@pytest.fixture()
+def descriptions():
+    return {
+        'descriptions': [{
+            'description': 'Hi I\'m a abstract',
+            'descriptionType': 'Abstract'
+        }, {
+            'description': 'Hi I\'m technical info',
+            'descriptionType': 'TechnicalInfo'
+        }]
+    }
+
+@pytest.fixture()
+def funding_references():
+    return {
+        'fundingReferences': [{
+            'funderName': 'Laura and John Arnold Foundation',
+            'funderIdentifier': {
+                'funderIdentifier': '<Crossref Funder ID> or whatever',
+                'funderIdentifierType': 'Crossref Funder ID',
+            },
+            "awardTitle": 'Reproducibility Project: Cancer Biology'
+        }]
+    }
+
+@pytest.fixture()
+def resource_type():
+    return {
+        'resourceType': {
+            'resourceType': 'something user specified',
+            'resourceTypeGeneral': 'Image'
+        }
+    }
+
+@pytest.fixture()
+def related_identifiers():
+    return {
+        'relatedIdentifiers': [{
+            'relatedIdentifier': 'something user specified',
+            'relatedIdentifierType': 'DOI',
+            'relationType': 'IsCitedBy'
+        }]
+    }
+
+@pytest.fixture()
+def misc_metadata():
+    # These requires little validation so they will be lumped together.
+    return {
+        'language': 'something user specified',
+        'sizes': ['something', 'user', 'specified'],
+        'formats': ['something', 'user', 'specified'],
+    }
+
+
+@pytest.mark.django_db
```

Comment:
@Johnetordoff, minor drive-by comment, you probably don't need this marker as none of the tests is using the db.